doc/recipe_elements/keyboard.rst

```diff
@@ -0,0 +1,99 @@
+==============
+Keyboard spoke
+==============
+
+/installation/hub/keyboard/add_layout
+=====================================
+Handles *add* (*+*) button and resulting dialog.
+
+Attributes:
+
+* ``name`` - name of the layout, e. g. ``cz`` or ``de (deadgraveacute)``
```

Comment:
I'll need to re-check this, but IIRC the 'raw' layout name that is used as identifier is not necessarily the same as what you see in Anaconda. However in such case I should mention this in the documentation to avoid confusion.

Reply:
A note about the correct name format would in my opinion be helpful. Or if the name should be taken from some non-obvious location it should be also said here to avoid misunderstandings, at least in my opinion.

Reply:
Solved.